specs/interop/shared-lockbox-upgrade.md

```diff
@@ -17,13 +17,12 @@
 
 ## Overview
 
-Based on the assumption that a chain joining the dependency set is an irreversible process,
-the on-chain chains list is simplified by assuming that joining the Shared Lockbox is
-equivalent to joining the op-governed dependency set.
+Based on the assumption that a chain joining the op-governed dependency set is an irreversible process,
+it is assume that joining the Shared Lockbox is equivalent to it.
```

Comment:
Is this a business decision or a protocol specification. Should we mention this here 🤔?
In which way is it helpful to explain that when talking about the upgrade process?